.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - node
+cache:
+  yarn: true
+  directories:
+    - website/node_modules
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+script:
+  - cd website
+  - yarn
```

Comment:
I don't know if it makes any difference, but I think it would make sense to separate dependency installation into `install` section rather than part of `script`.